<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>521</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QFrame" name="titleFrame">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>520</width>
        <height>80</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:1.916, fx:0.5, fy:0.5, stop:0.0743802 rgba(0, 0, 225, 255), stop:0.917355 rgba(255, 255, 255, 255));</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>20</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);
font: 28pt &quot;Tw Cen MT Condensed Extra Bold&quot;;</string>
       </property>
       <property name="text">
        <string>Tic Tac Toe</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>100</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(17, 21, 255);
font: 18pt &quot;Tw Cen MT Condensed Extra Bold&quot;;</string>
      </property>
      <property name="text">
       <string>Choose Option to Play</string>
      </property>
     </widget>
     <widget class="QPushButton" name="x_Btn">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>300</y>
        <width>80</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(17, 21, 255);
font: 22pt &quot;Tw Cen MT Condensed Extra Bold&quot;;</string>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
     <widget class="QPushButton" name="o_Btn">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>300</y>
        <width>80</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(17, 21, 255);
font: 22pt &quot;Tw Cen MT Condensed Extra Bold&quot;;</string>
      </property>
      <property name="text">
       <string>O</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>70</y>
        <width>291</width>
        <height>291</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 248, 172);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QPushButton" name="Btn0">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>91</width>
         <height>91</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(17, 21, 255);
font: 12pt &quot;Tw Cen MT Condensed Extra Bold&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="Btn1">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>0</y>
         <width>91</width>
         <height>91</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(17, 21, 255);
font: 12pt &quot;Tw Cen MT Condensed Extra Bold&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="Btn2">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>0</y>
         <width>91</width>
         <height>91</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(17, 21, 255);
font: 12pt &quot;Tw Cen MT Condensed Extra Bold&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="Btn4">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>100</y>
         <width>91</width>
         <height>91</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(17, 21, 255);
font: 12pt &quot;Tw Cen MT Condensed Extra Bold&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="Btn5">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>100</y>
         <width>91</width>
         <height>91</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(17, 21, 255);
font: 12pt &quot;Tw Cen MT Condensed Extra Bold&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="Btn3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>100</y>
         <width>91</width>
         <height>91</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(17, 21, 255);
font: 12pt &quot;Tw Cen MT Condensed Extra Bold&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="Btn7">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>200</y>
         <width>91</width>
         <height>91</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(17, 21, 255);
font: 12pt &quot;Tw Cen MT Condensed Extra Bold&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="Btn8">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>200</y>
         <width>91</width>
         <height>91</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(17, 21, 255);
font: 12pt &quot;Tw Cen MT Condensed Extra Bold&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="Btn6">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>200</y>
         <width>91</width>
         <height>91</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(17, 21, 255);
font: 12pt &quot;Tw Cen MT Condensed Extra Bold&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(17, 21, 255);
font: 12pt &quot;Tw Cen MT Condensed Extra Bold&quot;;</string>
      </property>
      <property name="text">
       <string>Player 1:   X</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>10</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(17, 21, 255);

font: 12pt &quot;Tw Cen MT Condensed Extra Bold&quot;;</string>
      </property>
      <property name="text">
       <string>Player 2:  O</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>390</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(17, 21, 255);
font: 12pt &quot;Tw Cen MT Condensed Extra Bold&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>385</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(17, 21, 255);
font: 12pt &quot;Tw Cen MT Condensed Extra Bold&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>529</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
